youtube_dl/extractor/cloudflarestream.py

```diff
@@ -11,7 +11,7 @@ class CloudflareStreamIE(InfoExtractor):
                     https?://
                         (?:
                             (?:watch\.)?cloudflarestream\.com/|
```

Comment:
watch.videodelivery.net should also be added.

Reply:
Ok.